lab-scott/test/test-greet.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const assert = require('assert');
+const greet = require(__dirname + '/../lib/greet.js');
+
+describe('Testing module lib/greet.js', function() {
+  describe('Testing function greeting; no cli arg', function() {
+    it('Should return Hello Nobody', function() {
+      assert.equal(greet(process.argv[2]), 'Hello Nobody');
+    });
+  });
+
+  describe('Testing function greeting; one cli arg', function() {
+    it('Should return Hello Scott', function() {
+      assert.equal(greet(process.argv[2] = 'Scott'), 'Hello Scott');
+    });
+  });
+
+  describe('Testing function greeting; multiple cli args', function() {
```

Comment:
@slugbyte Is this a valid test? It passes, but I can't figure/find the correct way to pass multiple args into `process.argv`. 

I removed the test from the lab02 assignment because this test was failing. Not sure if that's a TravisCI issue or an issue with the test itself.